watertap/examples/flowsheets/lsrro/lsrro.py

```diff
@@ -950,10 +959,18 @@ def optimize_set_up(
     for idx, stage in m.fs.ROUnits.items():
         stage.area.unfix()
         stage.width.unfix()
-        stage.area.setlb(1)
-        stage.area.setub(20000)
-        stage.width.setlb(0.1)
-        stage.width.setub(1000)
+        if not set_default_bounds_on_module_dimensions:
+            stage.area.setlb(1)
+            stage.area.setub(None)
+            stage.width.setlb(1)
+            stage.width.setub(None)
+            stage.length.setlb(1)
+        else:
+            # bounds originally set for Cost Optimization of LSRRO paper
+            stage.area.setlb(1)
+            stage.area.setub(20000)
+            stage.width.setlb(0.1)
+            stage.width.setub(1000)
```

Comment:
```suggestion
        if set_default_bounds_on_module_dimensions:
            # bounds originally set for Cost Optimization of LSRRO paper
            stage.area.setlb(1)
            stage.area.setub(20000)
            stage.width.setlb(0.1)
            stage.width.setub(1000)
        else:
            stage.area.setlb(1)
            stage.area.setub(None)
            stage.width.setlb(1)
            stage.width.setub(None)
            stage.length.setlb(1)
```

Reply:
No prob- committing your suggestion. Just curious- any niche cases where it would matter? Or is it more of a preference?

Reply:
It's a readability thing, nothing more.